grails-app/conf/Config.groovy

```diff
@@ -152,7 +151,7 @@ environments {
 
     production {
 
-        // URLs
+        // overwritten by Chef in production
         grails.serverURL = "http://myaodn.example.com"
         geonetwork.url = "http://catalogue-123.aodn.org.au/geonetwork"
```

Comment:
lets point this to catalogue-portal. I know it redirects, but we're not using this domain name anymore